### PR TITLE
updateLink for transient entities

### PR DESCRIPTION
**Pull request by HeikoTheissen**

### CODE REVIEW

**Review by ralfhandl — DISMISSED** [review]

File: vocabularies/Analytics.xml [hunk 1]
```diff
@@ -167,6 +167,44 @@
         </Property>
       </ComplexType>
 
+      <Term Name="UpdateAction" Type="Analytics.UpdateActionType" Nullable="false">
+        <Annotation Term="Common.Experimental" />
+        <Annotation Term="Common.IsInstanceAnnotation" />
+        <Annotation Term="Core.Description" String="Action for updating an aggregate entity" />
+        <Annotation Term="Core.LongDescription">
+          <String>Transient entities that result from the application of a transformation sequence represent an
+          aggregate of persistent entities (for example, aggregate sales per country and product, as in [OData-Aggr, example 19]).
```
Comment (ralfhandl) [inline vocabularies/Analytics.xml:176]: Will `[OData-Aggr, example 19]` become a real link once the next spec draft is out?

File: vocabularies/Analytics.xml [hunk 2]
```diff
@@ -167,6 +167,44 @@
...
+              </Record>
+            </Annotation>
+          </Record>
+        </Annotation>
+      </Term>
+      <ComplexType Name="UpdateActionType">
+        <Annotation Term="Common.Experimental" />
+        <Property Name="ActionURL" Type="Edm.String" Nullable="false">
+          <Annotation Term="Core.IsURL" />
+          <Annotation Term="Core.Description" String="Action URL [OData-Protocol, section 11.5.5]" />
```
Comment (ralfhandl) [inline vocabularies/Analytics.xml:197]: Please make this a real link to the stable section bookmark
```suggestion
          <Annotation Term="Core.Description" String="Action URL [OData-Protocol, section 11.5.5](https://docs.oasis-open.org/odata/odata/v4.01/odata-v4.01-part1-protocol.html#sec_Actions)" />
```

**Review by ralfhandl — CHANGES_REQUESTED** [review]

File: vocabularies/Analytics.xml (hunk 1 above)
Comment (HeikoTheissen) [reply vocabularies/Analytics.xml:176]: Yes.

File: vocabularies/Analytics.xml [hunk 3]
```diff
@@ -167,6 +167,44 @@
...
+          to update the transient entity. This effectively means updating the persistent entities whose aggregate
+          the transient entity represents, how this dis-aggregation happens is defined by the service.</String>
+        </Annotation>
+        <Annotation Term="Core.Example">
+          <Record>
+            <PropertyValue Property="Description" String="See [example model](../examples/Analytics.UpdateAction-sample.xml)" />
+            <Annotation Term="Analytics.UpdateAction">
+              <Record>
+                <PropertyValue Property="ActionURL" String="/UpdateAggregateSales" />
+                <PropertyValue Property="ActionParameter" String="groupby((Customer/Country,Product/Name),aggregate(Amount with sum as Total))/filter(Customer/Country eq 'Netherlands' and Product/Name eq 'Paper')" />
```
[the remark this answers is not in the document]
Comment (ralfhandl) [reply vocabularies/Analytics.xml:187]: How would we model this if we limit ourselves to custom aggregates? Which metadata annotations and which instance annotations would we use?

File: vocabularies/Analytics.xml [hunk 4]
```diff
@@ -175,6 +213,38 @@
           </Collection>
         </Annotation>
       </Annotations>
+      <Term Name="CustomDisaggregate" Type="Analytics.CustomDisaggregateType" Nullable="false" BaseTerm="Aggregation.CustomAggregate" AppliesTo="Collection EntityType">
```
Comment (HeikoTheissen) [inline vocabularies/Analytics.xml:216]: This allows one dis-aggregation rule per custom aggregate. Sufficient?

**Review by ralfhandl — APPROVED** [review]

**Comment by HeikoTheissen** [comment]
Rebase to main, see #212

**Pull request closed without merging**